src/mmrelay/cli.py

```
@@ -1493,6 +1748,136 @@ def handle_service_command(args: argparse.Namespace) -> int:
+                    )
+                    print(
+                        "        --from-literal=MMRELAY_MATRIX_PASSWORD="
+                        f"$MMRELAY_MATRIX_PASSWORD --namespace={config['namespace']}"
+                    )
+
+            print()
+            print("   2. Apply the manifests:")
+            print(f"      kubectl apply -f {output_dir}/")
+            print("\n📖 For detailed instructions, see docs/KUBERNETES.md")
+
+            return 0
```

Ruff (0.14.13)

Consider moving this statement to an `else` block

(TRY300)